Source/Core/DolphinQt2/ToolBar.cpp

```diff
@@ -91,15 +92,19 @@ void ToolBar::MakeActions()
 
 void ToolBar::UpdateIcons()
 {
-  QString dir = Settings().GetThemeDir();
-
-  m_open_action->setIcon(QIcon(QStringLiteral("open.png").prepend(dir)));
-  m_paths_action->setIcon(QIcon(QStringLiteral("browse.png").prepend(dir)));
-  m_play_action->setIcon(QIcon(QStringLiteral("play.png").prepend(dir)));
-  m_pause_action->setIcon(QIcon(QStringLiteral("pause.png").prepend(dir)));
-  m_stop_action->setIcon(QIcon(QStringLiteral("stop.png").prepend(dir)));
-  m_fullscreen_action->setIcon(QIcon(QStringLiteral("fullscreen.png").prepend(dir)));
-  m_screenshot_action->setIcon(QIcon(QStringLiteral("screenshot.png").prepend(dir)));
-  m_config_action->setIcon(QIcon(QStringLiteral("config.png").prepend(dir)));
-  m_controllers_action->setIcon(QIcon(QStringLiteral("classic.png").prepend(dir)));
+  m_open_action->setIcon(QIcon(Resources::GetScaledPixmap("open", 4, Resources::Directory::Theme)));
```

Comment:
Can't see the issue tbh honest, tried at different scales and couldn't notice any negative side effects, definitely not *worse than how it currently is*. Maybe if you could elaborate and suggest a way to fix that, until then I'm unable to address this.